solutions/ha-nva/layer-7-ingress/azuredeploy.json

```diff
@@ -0,0 +1,801 @@
+                            "direction": "Inbound"
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "condition": "[equals(parameters('DeployAppGatewayDmz'), 'Yes')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2019-10-01",
+            "name": "ParameterizedAppGateway",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/publicIPAddresses', 'pip-appgateway-hanva')]",
+                "[resourceId('Microsoft.Network/virtualNetworks', parameters('VirtualNetworkName'))]"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[uri(parameters('_artifactsLocation'), concat('nested/paramappgateway.json', parameters('_artifactsLocationSasToken')))]"
+                },
+                "parameters": {
+                    "location": {
+                        "value": "[parameters('location')]"
+                    },
+                    "appgateway_name": {
+                        "value": "ag-dmz"
+                    },
+                    "public_ip": {
+                        "value": "[resourceId('Microsoft.Network/publicIPAddresses', 'pip-appgateway-hanva')]"
+                    },
+                    "vnet_name": {
+                        "value": "[parameters('VirtualNetworkName')]"
+                    },
+                    "vnet_subnet_name": {
+                        "value": "ApplicationGatewaySubnet"
+                    }
+                }
+            }
+        },
+        {
+            "condition": "[equals(parameters('DeployLoadBalancerWeb'), 'Yes')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2019-10-01",
+            "name": "ParameterizedLoadBalancer",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualNetworks', parameters('VirtualNetworkName'))]"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[uri(parameters('_artifactsLocation'), concat('nested/paramlb.json', parameters('_artifactsLocationSasToken')))]"
+                },
+                "parameters": {
+                    "location": {
+                        "value": "[parameters('location')]"
+                    },
+                    "loadbalancer_name": {
+                        "value": "lb-web"
+                    },
+                    "vnet_name": {
+                        "value": "[parameters('VirtualNetworkName')]"
+                    },
+                    "vnet_subnet_name": {
+                        "value": "snet-web"
+                    }
+                }
+            }
+        },
+        {
+            "type": "Microsoft.KeyVault/vaults",
+            "name": "[concat('kv-hanva-',uniqueString(resourceGroup().id))]",
```

Comment:
Super nit, move name below api version for consistency and alignment with quickstart repo guidance.

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

Reply:
Fixed